contents/swift/welcome-to-swift.md

```diff
@@ -102,7 +105,7 @@ To learn more about Optionals, such as Optional Chaining and "dangerously" Force
 
 If you would like to seek a second (or more) opinion about Optionals, check out this [StackOverflow answer](http://stackoverflow.com/questions/24003642/what-is-an-optional-value-in-swift).
```

Comment:
Why dont you say something like:

If you would like to read up more about optionals, check these links out: 

or

If you want to know more about optionals, have a look at this article.